projects/wave-core/src/lib/util/pipes/highlight.pipe.ts

```diff
@@ -1,22 +1,19 @@
-import {Pipe, PipeTransform} from '@angular/core';
+import { PipeTransform, Pipe } from '@angular/core';
 
 @Pipe({name: 'waveHighlightPipe'})
-export class HighlightPipe implements PipeTransform {
 
-    // TODO: replace <span> with a <higlight> component?
-    transform(text: string, term: string): string {
+export class HighlightPipe implements PipeTransform {
+    transform(text: string, search): string {
```

Comment:
```suggestion
    transform(text: string, searchTerm: string): string {
```
You stripped the data type of the search term, but I guess it is always a string.

Reply:
does not exist anymore in this form